openmc/plots.py

```diff
@@ -634,19 +637,23 @@ def to_xml_element(self):
                 color = _SVG_COLORS[color.lower()]
             subelement.text = ' '.join(str(x) for x in color)
 
+        # Helper function to handle either int or Cell/Material
+        def get_id(domain):
+            return getattr(domain, 'id', domain)
```

Comment:
A few thoughts combined into one here:

  - we may as well make the comment above part of the docstring for the method
  - prefix the method name with `_` if we're only using it within the `Plot` class
  - make the behavior of this function more explicit. It may be obvious to some but it took me some staring to figure out what that call was doing.

```suggestion
        def _get_id(domain):
        """Helper function to handle either int or Cell/Material"""
            return domain is isinstance(domain, Integral) else domain.id
```

Reply:
This function is defined _inside_ of the `to_xml_element` method, so it is a local variable that is not visible to the outside world. That's why I didn't bother with a docstring (because it will never be viewed) or the leading underscore (it's not a method). I agree the use of `getattr` could be confusing so I'll go ahead and change that and provide a better comment.

Reply:
Ah, of course. Missed that. Sorry. Updates look good to me!